climada/entity/impact_funcs/base.py

```diff
@@ -273,3 +273,74 @@ def set_sigmoid_impf(self, *args, **kwargs):
         LOGGER.warning("The use of ImpactFunc.set_sigmoid_impf is deprecated."
                        "Use ImpactFunc.from_sigmoid_impf instead.")
         self.__dict__ = ImpactFunc.from_sigmoid_impf(*args, **kwargs).__dict__
+
+    @classmethod
+    def from_poly_s_shape(
+        cls,
+        intensity: tuple[float, float, int],
+        threshold: float,
+        half_point: float,
+        scale: float,
+        exponent: float,
+        haz_type: str,
+        impf_id: int = 1,
+        **kwargs):
+        """S-shape polynomial impact function hinging on four parameter.
+
+        .. math:: f(I) = \\frac{luk(I)**exponent}{1 + luk(I)**exponent}}\\cdot scale
+
+        .. math:: luk(I) = \\frac{max[I - threshold, 0]}{half_point - threshold}
```

Comment:
Please write valid LaTeX. You don't have to excape characters here. You can use a single math directive for both formulae. Please use `\max` and either use `\mathrm{threshold}` or a single symbol, as in any math formula.